web/static/js/contributors.js

```diff
@@ -1,5 +1,29 @@
-$(document).ready(function(){
-  jQuery.get('https://api.github.com/repos/codethesaurus/codethesaur.us/contributors?accept=application/vnd.github.v3+json&per_page=100&anon=1', function(data, status){
-    $("#contributors").text(data.length)
-  })
-})
\ No newline at end of file
+document.addEventListener("DOMContentLoaded", function () {
+  var contributorsRequest = new XMLHttpRequest();
+
+  contributorsRequest.open(
+    "GET",
+    "https://api.github.com/repos/codethesaurus/codethesaur.us/contributors?accept=application/vd.github.v3+json&per_page=200&anon=1"
+  );
+
+  contributorsRequest.send();
+
+  contributorsRequest.onload = function () {
+    if (contributorsRequest.status != 200) {
+      document.querySelector("#contributors").innerHTML = 0;
```

Comment:
Hmm, wonder if we could show a better error. Maybe hide the overarching element altogether? 0 contributors will look kind of weird

Reply:
Presently if the call never happens, I think it just shows "and contributors." Maybe it could get changed to "and multiple contributors" or something like that?

![image](https://user-images.githubusercontent.com/2601974/165202560-d4392923-ab43-4d4e-978f-e50f93790bf3.png)

Reply:
i changed it to multiple contributors, it's better than zero indeed.